### PR TITLE
Add example inventory file

### PR DESCRIPTION
**Pull request by mbukatov**
This pull request adds an example inventory file.

This is work in progress, feedback welcome.

Implementing https://github.com/Tendrl/tendrl-ansible/issues/84

### CODE REVIEW

**Comment by julienlim** [comment]
@mbukatov I like the sample host inventory file.  We could probably improve a little more on the comments, and for inspiration take a look at https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example.  I like the way they section of the mandatory vars and the optional/additional vars to make it a little easier to read and know what's required or not.

Also, it may be a good idea to update the readme to mention this (hosts.example) is an example of  inventory_file referenced in the installation docs.

Thanks for the quick turnaround!

**Review by julienlim — APPROVED** [review]
Looks good to me.

**Comment by mbukatov** [comment]
Looking at the openshift-ansible roles and documentation, they don't have much details related to variables in role readme files so they don't have to deal with duplication of information there.

In tendrl-ansible, we need to address this duplication first, as we have description of ansible variables in role readme files, which matches what people usually do when maintaining roles in ansible galaxy.

Btw there seems to be [no consensus about documenting roles in upstream ansible](https://github.com/ansible/proposals/issues/19), so no matter what we do, some people will find it weird.

**Comment by julienlim** [comment]
Ack @mbukatov.  With the example file (with the inventory + vars), it's a lot easier, and I created my own when I did a recent tendrl-ansible deployment, and it was very easy to get started.  A lot easier than before when I had to go hunting in the site.yml file!

I hope we can merge this as like you said as I think there's more value than not than waiting to try and solve every combination -- even if we say this is an example for POCs.

**Comment by mbukatov** [comment]
@julienlim I'm thinking about minimal changes to add into the example host file to make it clear that not every option possible is here. One problem with including a minimal example like I have suggested is that someone can miss that ...

**Comment by mbukatov** [comment]
If the feedback is positive, I'm going to merge it in the current state. Long term solution would include de-emphasizing description of variables in role readme files, moving the main description into main readme file and the host file example.

File: hosts.example [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This is an example of minimal tendrl-ansible host inventory file, aligned
+# with instructions in README.md file. For description of installation steps
+# and full list of ansible variables, see the README.md file and readme files
+# of tendrl-ansible roles.
+# This example includes 4 *Tendrl Storage Nodes* (GlusterFS servers) and one
```
Comment (dahorak) [inline hosts.example:5]: I'm not sure, how we are referring the Storage nodes on other places, but *"Gluster Storage Servers" (Monitored by Tendrl)* would sound slightly more clear to me. In the meaning, that the storage servers are primarily ordinary "Gluster Storage servers" and the fact that they are/will be monitored by Tendrl is only additional information.

File: hosts.example [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# tendrl.example.com hostname is translated to 192.0.2.1 ip address.
+etcd_ip_address=192.0.2.1
+etcd_fqdn=tendrl.example.com
+graphite_fqdn=tendrl.example.com
+
+# Uncomment and tweak the following options when direct ssh login of root user
+# is not allowed and you are connecting via non-root cloud-user account, which
+# can leverage sudo to run any command as root without any password.
+#ansible_become=yes
+#ansible_user=cloud-user
```
Comment (dahorak) [inline hosts.example:30]: What about some small note like this?
```
# Place for any additional (optional) configuration variables described in readme files
# of tendrl-ansible roles.
```

**Review by dahorak — APPROVED** [review]
Just two small notes (suggestions), but overall looks good.

**Review by julienlim — APPROVED** [review]
Looks good to me.

File: hosts.example (hunk 1 above)
Comment (mbukatov) [reply hosts.example:5]: Good point.

Comment (mbukatov) [reply hosts.example:5]: That said, in the README, we have:

>  `tendrl-ansible.tendrl-storage-node`: installation of **Tendrl Storage Node**

File: hosts.example (hunk 2 above)
Comment (mbukatov) [reply hosts.example:30]: Good idea.

**Pull request merged**